docs/get-started/nextjs.md

```diff
@@ -0,0 +1,921 @@
+
+## Sass/Scss
+
+[Global sass/scss stylesheets](https://nextjs.org/docs/pages/building-your-application/styling/sass) are supported without any additional configuration as well. Just import them into [`.storybook/preview.js|ts`](../configure/index.md#configure-story-rendering)
+
+```js
+// .storybook/preview.js|ts
+import '../styles/globals.scss';
+```
+
+This will automatically include any of your [custom sass configurations](https://nextjs.org/docs/pages/building-your-application/styling/sass#customizing-sass-options) in your `next.config.js` file.
+
+```js
+// next.config.js
+import * as path from 'path';
+
+export default {
+  // Any options here are included in Sass compilation for your stories
+  sassOptions: {
+    includePaths: [path.join(__dirname, 'styles')],
+  },
+};
+```
+
+## Css/Sass/Scss Modules
+
+[css modules](https://nextjs.org/docs/pages/building-your-application/styling/css-modules) work as expected.
+
+```jsx
+// This import will work in Storybook
+import styles from './Button.module.css';
+// sass/scss is also supported
+// import styles from './Button.module.scss'
+// import styles from './Button.module.sass'
+
+export function Button() {
+  return (
+    <button type="button" className={styles.error}>
+      Destroy
+    </button>
+  );
+}
+```
+
+## Styled JSX
+
+The built in CSS-in-JS solution for Next.js is [styled-jsx](https://nextjs.org/docs/pages/building-your-application/styling/css-in-js), and this framework supports that out of the box too, zero config.
+
+```jsx
+// This will work in Storybook
+function HelloWorld() {
+  return (
+    <div>
+      Hello world
+      <p>scoped!</p>
+      <style jsx>{`
+        p {
+          color: blue;
+        }
+        div {
+          background: red;
+        }
+        @media (max-width: 600px) {
+          div {
+            background: blue;
+          }
+        }
+      `}</style>
+      <style global jsx>{`
+        body {
+          background: black;
+        }
+      `}</style>
+    </div>
+  );
+}
+
+export default HelloWorld;
+```
+
+You can use your own babel config too. This is an example of how you can customize styled-jsx.
+
+```json
+// .babelrc or whatever config file you use
+{
+  "presets": [
+    [
+      "next/babel",
+      {
+        "styled-jsx": {
+          "plugins": ["@styled-jsx/plugin-sass"]
+        }
+      }
+    ]
+  ]
+}
+```
+
+## Postcss
+
+Next.js lets you [customize postcss config](https://nextjs.org/docs/pages/building-your-application/configuring/post-css). Thus this framework will automatically handle your postcss config for you.
+
+This allows for cool things like zero config tailwindcss! (See [Next.js' example](https://github.com/vercel/next.js/tree/canary/examples/with-tailwindcss))
+
+## Absolute Imports
+
+[Absolute imports](https://nextjs.org/docs/pages/building-your-application/configuring/absolute-imports-and-module-aliases#absolute-imports) from the root directory are supported.
+
+```jsx
```

Comment:
When you're able can you add a comment referencing the filename so that it depicts the filename accurately? Optionally, this could be converted into two examples one for JavaScript and one for TypeScript